src/components/scenes/WcConnectScene.tsx

```diff
@@ -27,6 +27,10 @@ import { SelectableRow } from '../themed/SelectableRow'
 
 interface Props extends EdgeSceneProps<'wcConnect'> {}
 
+export interface WcConnectSceneParams {
```

Comment:
The word "Scene" should not be part of these type names. We've been using `XyzScene` / `XyzParams`, not `XyzSceneParams`.